styles/aboutUs.scss

```diff
@@ -3,20 +3,25 @@
   width: 100%;
   width: 100vw;
 
-  padding: $section-spc;
+  padding: $section-mobile-spc;
```

Comment:
I wouldn't want to define spacing variables depending on the device or screen size. Let's keep them general.